Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -40,9 +43,9 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(Configuration)' != 'Release'">
-    <InternalsVisibleTo Include="Snowflake.Data.Tests" />
+    <InternalsVisibleTo Include="Snowflake.Data.Tests,PublicKey=0024000004800000940000000602000000240000525341310004000001000100c159643cbf31e9bbbe7920091c39ecbd988b3e102df664830b33f12c9d838b07463682948cb56bbc414311fa019d6bca1dc08bc9d54684667fdd94ddf140f499ccb66795322128ba1804d8342fec912cea2704fe09c82dbf03bf10817330537d57e8177d92c205e93278245d18a84f185c53d1c5d917171c9807ac769ae9dfcb" />

```

Comment:
Snowflake.Data.Tests is not released so we don't need to sign it.

Reply:
This is required for testing